src/index.tsx

```diff
@@ -1,19 +1,24 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import React from "react"
+import ReactDOM from "react-dom/client"
+import "./index.css"
+import App from "./App"
 
-const root = ReactDOM.createRoot(
-  document.getElementById('root') as HTMLElement
-);
+import {BrowserRouter} from "react-router-dom"
+
+import {UserProvider} from "./context/user.provider"
+
+const root = ReactDOM.createRoot(document.getElementById("root") as HTMLElement)
 root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
-);
+	// <React.StrictMode>
```

Comment:
Remove commented code.

Reply:
Uncommented this.

I commented this for debugging.